packages/siop-oid4vp/lib/authorization-response/PresentationExchange.ts

```diff
@@ -76,8 +82,11 @@ export class PresentationExchange {
         keyEncoding: options?.signatureOptions?.keyEncoding ?? KeyEncoding.Hex,
       },
     }
+    return await this.pex.verifiablePresentationFrom(presentationDefinition, this.removeMDocCredentials(selectedCredentials), presentationSignCallback, signOptions)
```

Comment:
Please make comments when you make these types of exceptions as to why it is needed

Reply:
```
    // When there are MDoc credentials among the selected ones, filter those out as pex does not support mdoc credentials
    const filteredCredentials = this.removeMDocCredentials(selectedCredentials) 
    return await this.pex.verifiablePresentationFrom(presentationDefinition, filteredCredentials, presentationSignCallback, signOptions)
```